PUSH_TO_FILE.md

```diff
@@ -709,24 +710,69 @@ annotation-based configuration and/or push-to-file mode:
 ```
-Test patch against a live deployment:
+- Find the pod name
 ```
-kubectl patch deployment \$DEPLOYMENT_NAME --type json --patch-file patch.json --dry-run=server
+kubectl get pods
 ```
-Preview the new deployment:
+- Display the logs
 ```
-kubectl patch deployment \$DEPLOYMENT_NAME --type json --patch-file patch.json --dry-run=server --output yaml
+kubectl logs <pod-name> -c <init-container-name>
 ```
-Apply patch:
+To display the Secrets Provider logs in Openshift.
+
+- Go to the app namespace
 ```
-kubectl patch deployment \$DEPLOYMENT_NAME --type json --patch-file patch.json
+oc project <project-name>
 ```
+- Find the pod name
+```
+oc get pods
+```
+- Display the logs
+```
+oc logs <pod-name> -c <init-container-name>
+```
+
+### Common issues displayed in the logs and resolutions
+
+|  Issue      |       Error code   | Resolution  |
+| ----------- | ------------------ | ------ |
+| Secrets Provider for Kubernetes failed to update secrets in file mode | `CSPFK039E` | Check that the secret key pairs have been configured correctly.  |
+| Failed to open annotations file                                       | `CSPFK041E` | The annotations file cannot be opened by the Secrets provider. Check the volume mounts and downward API configuration.  |
+| Annotation 'x' does not accept value                                  | `CSPFK042E` | The annotation has a key and a value. Here the is value invalid. Check the value for the annotation that is displayed in the error message.     |
+| Annotation 'x' does not accept value                                  | `CSPFK043E` | This is similar to CSPFK042E.  Check the value for the annotation that is displayed in the error message.      |
+| Annotation file line 'x' is malformed:                                | `CSPFK045E` | The annotation described in the message is malformed. Check the annotation has a valid key and value       |
+| Secret Store Type needs to be configured                              | `CSPFK046E` | Secret Store Type or secrets-destination  has not been configured with `conjur.org/secrets-destination`. If the secrets-destination is set via annotations verify that the downward API volumes are configured correctly.       |
+| Secrets Provider in Push-to-File mode can only be configured with Pod annotations | `CSPFK047E`| To configure push-to-file the Secrets store type must have the `conjur.org/secrets-destination` with pod annotations and cannot be set with the `SECRETS_DESTINATION` environment variable. If the secrets-destination is set via annotations verify that the pod volumes and volumeMounts are configured correctly.|
+| Secrets Provider in K8s Secrets mode requires either the 'K8S_SECRETS' environment variable or 'conjur.org/k8s-secrets' | `CSPFK048E `        | If the `secrets-destination` is set to `k8s_secrets` then the 'K8S_SECRETS' environment variable or 'conjur.org/k8s-secrets'. This does not apply to push-to-file.       |
```

Comment:
> If the `secrets-destination` is set to `k8s_secrets` then the 'K8S_SECRETS' environment variable or 'conjur.org/k8s-secrets'.`

Are there missing words in this sentence?

Reply:
yes, thanks.